modules/manage/pages/manage-settings/general-settings.adoc

```diff
@@ -318,7 +318,10 @@ The default cleared setting has an Index Server node rebuild any newly assigned
 You cannot enable file-based rebalance when you have enabled Memory Optimized Index Storage.  
 When you select this option, Couchbase Server copies the index files from one Index Server node to another during a rebalance instead of rebuilding them. 
 See xref:learn:clusters-and-availability/rebalance.adoc#index-rebalance-methods[Index Rebalance Methods].
-
++
+You can deactivate this feature from the UI or via REST API. To learn about deactivating this feature via REST API, see xref:rest-api:post-settings-indexes.adoc#deactivate-file-transfer-based-rebalance[Curl Command to Deactivate the File Transfer Based Rebalance Setting]. This feature is inactive by default.
```

Comment:
```suggestion
You can disable this feature from the UI or via REST API. To learn about disabling this feature via REST API, see xref:rest-api:post-settings-indexes.adoc#deactivate-file-transfer-based-rebalance[Curl Command to Deactivate the File Transfer Based Rebalance Setting]. This feature is disabled by default.
```

Reply:
A tiny point, but I think we tend to use `disable` rather than `deactivate`.

Reply:
Yes, Sarah too asked to follow what's on the UI.